src/Monolog/Handler/StorageApiHandler.php

```diff
@@ -98,7 +98,11 @@ public function getVerbosity()
     protected function initStorageApiClient()
     {
         try {
-            $this->storageApiClient = $this->storageApiService->getClient();
+            $this->storageApiClient = $this->container->get('syrup.storage_api')->getClientWithoutLogger();
```

Comment:
to je kvůli té circular dependency, že? je to dobrej brainfuck :-)

Reply:
jop